### PR TITLE
docs(common-css): common css readme

### PR DESCRIPTION
**Pull request by ynnenu**
4. Documentation
- [x] Storybook documentation has been created/updated

### CODE REVIEW

File: packages/common-css/README.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
-# Fundamental Library Common CSS Package
+Fundamental Library Common CSS Package
 
-This package contains Common CSS files. More documentation to come soon...
+## What is Fundamental Library Common CSS?
+
+Fundamental Library Common CSS Package holds all the CSS style rules of the library that are recognized by older browsers—as well as the newer browsers. It creates the style rules for the basic elements on a page, such as a (link) elements, p (paragraph) elements, h1 (header) elements etc. The package is based on the DRY (Don't Repeat Yourself) methodology which allows the developers to not repeat the same CSS rules overall in the library.
+
```
Comment (g-cheishvili) [inline packages/common-css/README.md:6]: @InnaAtanasova Help us with description. Is this right?

File: packages/common-css/README.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+Fundamental Library Common CSS Package is part of the Fundamental Styles. If you have installed the Fundamental Styles in your project, you automatically get the Fundamental Library Common CSS Package.
+
+**Learn more at http://sap.github.io/fundamental-styles/**
+
+### 2. Fundamental Library Common CSS Package on its own
+
+Fundamental Library Common CSS Package can stand on its own. You can install the package in your project through:
+
+```html
```
Comment (g-cheishvili) [inline packages/common-css/README.md:19]:
```suggestion
```bash
```

File: packages/common-css/README.md (hunk 1 above)
Comment (mikerodonnell89) [reply packages/common-css/README.md:6]: I don't think we're backwards compatible throughout the history of web browsers so I'd remove the bit about older and newer browsers. And I think if we're going to mention DRY let's also mention BEM

File: packages/common-css/README.md [hunk 3]
```diff
@@ -1,3 +1,37 @@
...
+
+Fundamental Library Common CSS Package can stand on its own. You can install the package in your project through:
+
+```bash
+npm i @fundamental-styles/common-css
+```
+
+## Common CSS Elements
+
+The Fundamental Library Common CSS Package consists of elements that align with the SAP designs.
```
Comment (mikerodonnell89) [inline packages/common-css/README.md:25]:
```suggestion
## Common CSS Properties

The Fundamental Library Common CSS Package consists of property values that align with the SAP designs.
```

**Review by mikerodonnell89 — COMMENTED** [review]
Looks good! couple small comments

File: packages/common-css/README.md (hunk 1 above)
Comment (InnaAtanasova) [reply packages/common-css/README.md:6]: Fundamental Library Common CSS Package is a set of utility classes and SCSS mixins that allow developers to add padding, margin, color, font, shadow, and more to their applications and controls without writing custom css. 
The class naming follows the BEM methodology and the values of the CSS rules are based on SAP Fiori Design Guidelines. Further customization is possible by using the SCSS mixins and providing user defined values as parameters.

Comment (InnaAtanasova) [reply packages/common-css/README.md:6]: +1 on what Mike said

File: packages/common-css/README.md (hunk 3 above)
Comment (InnaAtanasova) [reply packages/common-css/README.md:25]: I think here we should mention it's SAP Fiori design

File: packages/common-css/README.md (hunk 1 above)
Comment (g-cheishvili) [reply packages/common-css/README.md:6]: @ynnenu what these two said 😁

**Review by platon-rov — COMMENTED** [review]
Seems like we have unified PR https://github.com/SAP/fundamental-styles/pull/4018 for all the readmes, should we then keep this one?

**Comment by ynnenu** [comment]
> 

This one is separated from the others. We decided to push all read me in one PR after I had created this one.

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**